mantis-control-plane/mantis-control-plane-server/src/main/java/io/mantisrx/server/master/MasterMain.java

```diff
@@ -365,6 +367,7 @@ public static void main(String[] args) {
         }
 
         try {
+            SpectatorRegistryFactory.setRegistry(new DefaultRegistry());
```

Comment:
why is this needed? Even if it's required, wouldn't it make sense to do this in nfmantis similar to worker-titus.

Reply:
If this is for test containers, I would recommend moving this there since this module is not expected to be used as the leaf node.

Reply:
in default masterMain-> main the spectator's default composite registry is empty thus no metrics is actually logged. This is needed for any app using this entry to be able to actually log metrics there.